app/settings/users/create.controller.js

```diff
@@ -30,44 +30,50 @@ function (
         $rootScope.$emit('setPageTitle', title);
     });
 
+    $scope.isValid = true;
```

Comment:
shouldn't isValid start assuming false? Just a question since I'm not sure if the repercusion is that it'll be weird in the UX if we change it